app/client/.gitignore

```diff
@@ -32,4 +32,11 @@ yarn-error.log*
 
 # vercel
 .vercel
-_unused
\ No newline at end of file
+
+_unused
+
+# playwright
+test-results/
+playwright-report/
+allure-report/
+allure-results/
```

Comment:
Should these also be added to the .dockerignore?

Reply:
Oh yes I think so, good catch.